### ISSUE TITLE
Can't download app via F-Droid

### ISSUE THREAD

**Issue by yujinio** [bfabiszewski/ulogger-android#133]
Having a Pixel 6a and the latest GrapheneOS installed, I can not download and install the app using any F-Droid client. 

The only option available to me at the application page is to share it.

**Comment by yujinio** [comment]
@bfabiszewski hey thank you for your prompt reply.

Yes, I am able to install other apps. 

I have just ensured, for some reason f-droid says the app is incompatible: `No versions compatible with device`, thus not allowing me to download it.

After I enabled a flag `Include incompatible versions`, the f-droid app allowed me to download ulogger.

**Comment by yujinio** [comment]
Update: incompatibility is cause by `Requires android.hardware.location.network`.

**Comment by bfabiszewski** [comment]
Good to know. I don't remember whether it is really needed in the manifest. Maybe I could add `android:required="false"`. This is rare case though. 
I'll check that. Thanks!

**Comment by yujinio** [comment]
@bfabiszewski I know it's definitely required that you state this requirement if you use cellular and wifi capabilities of a phone to get location data:
```
The app uses coarse location coordinates obtained from a network-based geolocation system supported on the device.

By using this feature, an app implies that it also uses the android.hardware.location feature, unless this parent feature is declared with the attribute android:required="false".
```

 so probably the problem is on the graphene os side, I will continue investigating it and will let you know what I got here

**Comment by ohhai** [comment]
> The application is available on F-Droid repo for Android 5.0 or newer.

Does it mean that it's compatible with Android 15?
I want to update to it, but I'd better wait if ulogger is not compatible.
